### ISSUE TITLE
Shape issue

### ISSUE THREAD

**Issue by relh** [arahusky/Tensorflow-Segmentation#6]
Hi, Great repo! I have succeeded at training a 128x128 segmentation net but am curious how to train a 250x250. If I just change the IMAGE_HEIGHT and IMAGE_WIDTH at the top to 250 and 250 I get this error:

Traceback (most recent call last):
  File "convolutional_autoencoder.py", line 342, in <module>
    train()
  File "convolutional_autoencoder.py", line 210, in train
    network = Network()
  File "convolutional_autoencoder.py", line 98, in __init__
    self.train_op = tf.train.AdamOptimizer().minimize(self.cost)
  File "/usr/lib/python3.4/dist-packages/tensorflow/python/training/optimizer.py", line 315, in minimize
    grad_loss=grad_loss)
  File "/usr/lib/python3.4/dist-packages/tensorflow/python/training/optimizer.py", line 386, in compute_gradients
    colocate_gradients_with_ops=colocate_gradients_with_ops)
  File "/usr/lib/python3.4/dist-packages/tensorflow/python/ops/gradients_impl.py", line 562, in gradients
    in_grad.set_shape(t_in.get_shape())
  File "/usr/lib/python3.4/dist-packages/tensorflow/python/framework/ops.py", line 378, in set_shape
    self._shape = self._shape.merge_with(shape)
  File "/usr/lib/python3.4/dist-packages/tensorflow/python/framework/tensor_shape.py", line 566, in merge_with
    raise ValueError("Shapes %s and %s are not compatible" % (self, other))
ValueError: Shapes (?, 125, 125, 64) and (?, 126, 126, 64) are not compatible

**Comment by arahusky** [comment]
Hello @relh , thanks for your feedback!

The issue you describe is similar to https://github.com/arahusky/Tensorflow-Segmentation/issues/2. 
TL;DR: current architecture does not work on 250x250, but works on 256x256, resize all images to that size when you want to train the network on images bigger than 128x128.
